zng/docs/spec.md

```diff
@@ -307,7 +332,11 @@ existing type ID ``<type-id>``.  ``<type-id>`` is encoded as a `uvarint` and `<n
 is encoded as a `uvarint` representing the length of the name in bytes,
 followed by that many bytes of UTF-8 string.
 
-### 2.1.2 End-of-stream Markers
+It is an error to define an alias that has the same name as a primitive type.
+It is also an error to redefine a previously defined alias with a
+type that differs from the original definition.
```

Comment:
I moved this up from the TZNG section, as it reads as if it's about ZNG in general.